File: MiniDOMOffice/CRM/CCampagnaCRM/AllowNegate/CCampagnaXUserAllowNegate.cs
using System;
using System.Collections;
using System.Collections.Generic;
using DMD;
using DMD.XML;
using DMD.Databases;
using DMD.Databases.Collections;
using minidom;
using minidom.repositories;
using static minidom.Sistema;
using static minidom.Anagrafica;
using static minidom.Office;

namespace minidom
{
    public partial class CustomerCalls
    {

        /// <summary>
        /// Autorizzazione o negazione esplicita per un utente
        /// </summary>
        [Serializable]
        public class CCampagnaXUserAllowNegate 
            : UserAllowNegate<CCampagnaCRM>
        {

            /// <summary>
            /// Costruttore
            /// </summary>
            public CCampagnaXUserAllowNegate()
            {
            }

            /// <summary>
            /// Repository
            /// </summary>
            /// <returns></returns>
            public override CModulesClass GetModule()
            {
                return null;
            }


            /// <summary>
            /// Discriminator
            /// </summary>
            /// <returns></returns>
            public override string GetTableName()
            {
                return null;
            }
 
            /// <summary>
            /// ID
            /// </summary>
            /// <returns></returns>
            protected override string GetItemFieldName()
            {
                return "";
            }
        }
    }
}

File: MiniDOMOffice/CRM/CCampagnaCRM/CCampagnaXUserAllowNegate.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.Databases

Imports minidom.Anagrafica
Imports minidom.XML

Partial Public Class CustomerCalls


    <Serializable>
    Public Class CCampagnaXUserAllowNegate
        Inherits UserAllowNegate(Of CCampagnaCRM)


        Public Sub New()
        End Sub

        Public Overrides Function GetModule() As CModule
            Return Nothing
        End Function

        Public Overrides Function GetTableName() As String
            Return Nothing
        End Function

        Protected Overrides Function GetConnection() As CDBConnection
            Return Nothing
        End Function

        Protected Overrides Function GetItemFieldName() As String
            Return ""
        End Function
    End Class



End Class